### PR TITLE
OSDOCS-3072: Adds RNs for user preferences and settings

### PR DESCRIPTION
**Pull request by opayne1**
[OSDOCS-3072](https://issues.redhat.com/browse/OSDOCS-3072)

- Applies to `enterprise-4.10` only
- [Preview link](https://deploy-preview-41543--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes.html#ocp-4-10-web-console)

### CODE REVIEW

**Comment by opayne1** [comment]
@kdoberst PTAL

**Review by zherman0 — APPROVED** [review]

**Comment by opayne1** [comment]
@TheRealJon PTAL

**Review by TheRealJon — APPROVED** [review]
/lgtm

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -122,6 +122,17 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
 [id="ocp-4-10-web-console"]
 === Web console
 
+==== Run a pod in debug mode
```
Comment (adellape) [inline release_notes/ocp-4-10-release-notes.adoc:125]: Maybe use a gerund here:

"Running a pod in debug mode"

or alternatively something like "Pod debug terminals"

Comment (adellape) [reply release_notes/ocp-4-10-release-notes.adoc:125]: Heading needs an ID/anchor here.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -122,6 +122,17 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
...
+With this update, you can now view debug terminals in the web console. When a pod has a container that is in a `CrashLoopBackOff` state, a debug pod can be launched. A terminal interface is provided and can be used to debug the crash looping container. This feature can be accessed by:
+
+* The pod status pop-over, which is accessed by clicking on the status of a pod, provides links to debug terminals for each crash looping container within that pod.
+* On the *Logs* tab of the pod details page, a debug terminal link will be provided above the log window when a crash looping container is selected.
+
+Additionally, the pod status pop-over now provides links to the *Logs* and *Events* tabs of the pod details page.
+
+==== User Preferences
```
[the remark this answers is not in the document]
Comment (adellape) [reply release_notes/ocp-4-10-release-notes.adoc:133]: Heading needs an ID/anchor here.

**Comment by yanpzhan** [comment]
The user preference part looks good to QE.

**Comment by XiyunZhao** [comment]
The content on the section of 'Running a pod in debug mode' is looks good to me, the description for the major function is correct and clearly

**Pull request merged**